lua/terrortown/lang/en.lua

```diff
@@ -2098,3 +2098,8 @@ L.builtin_marker = "Built-in."
 L.equipmenteditor_desc_builtin = "This equipment is built-in, it comes with TTT2!"
 L.help_roles_builtin = "This role is built-in, it comes with TTT2!"
 L.header_equipment_info = "Equipment information"
+
+-- 2023-12-24
+L.cl_sb_row_sresult_direct_conf = "Direct confirmation"
+L.cl_sb_row_sresult_in_conf = "Indirect confirmation"
```

Comment:
I know what you are atalking about, but this might be "programmer translation". Is this really understandable? Or should it be something like "confirmed through kill list"?

Reply:
I really don't know how to refer to these things, even in english it's a complicated thing that is a full phrase, and maybe that is what it should be instead. But players understand "confirmed" aspect.

Reply:
maybe "indirect confirmation" is the right call here, dunno. I just wanted to comment on this.

Reply:
i think if "indirect confirmation" is the text used to describe this we should just show no icon at all.

if i'm already wondering how a player ended up in the "confirmed dead" part of the scoreboard with no role and no other info attached, then "indirect confirmation" doesn't tell me anything more than i already know.

Reply:
![image](https://github.com/TTT-2/TTT2/assets/5711436/2eadd78e-9f0c-4f39-8faf-aa3bbcefbe9f)
![image](https://github.com/TTT-2/TTT2/assets/5711436/582cba06-3742-4657-a6fd-36685d0183c4)
I do agree, but "indirect confirmation" is still the general thing that links the minihud gray X to its role-less, confirmed position on the scoreboard. My thoughts are those be more closely related.

On the other hand, reserving the icons to EXCLUSIVELY mean that some level of bodysearch information is available will drastically cut down the time players spend folding and unfolding scoreboard data, and all the tooltips could be amended to say "Click to expand bodysearch info" or something. (if it were an imgbutton, right now it can't pass-through doclick)

The purpose was to bring the two headcount trackers closer together, but maybe the other changes are stronger without it, I don't know.

Reply:
Histalek is probably not aware of indirect confirmation. This happens when a killer is confirmed. Their kill list ist then used to confirm the other players as dead, without a role or any other information

Reply:
>  I do agree, but "indirect confirmation" is still the general thing that links the minihud gray X to its role-less, confirmed position on the scoreboard. My thoughts are those be more closely related.

In that case we should use the same icons for those, right? I think that would better be done in a scoreboard rework.

> On the other hand, reserving the icons to EXCLUSIVELY mean that some level of bodysearch information is available will drastically cut down the time players spend folding and unfolding scoreboard data, and all the tooltips could be amended to say "Click to expand bodysearch info" or something. (if it were an imgbutton, right now it can't pass-through doclick)

That's exactly what i would go for here.

> The purpose was to bring the two headcount trackers closer together, but maybe the other changes are stronger without it, I don't know.

I think bringing mini-scoreboard and scoreboard mroe closely together should really be done when we rework the scoreboard. 

> Histalek is probably not aware of indirect confirmation.

I'm aware of the kill-list, but having someone in the "confirmed dead" section of the scoreboard with no further bodysearch information already conveys that they were confirmed through a kill list. An additional icon is just superfluous in that case.

Reply:
we don't have icons for the minihud that match the spyglass or direct confirmed states i introduce here, and the minihud icons aren't really suited for the existing scoreboard 'cause they're single color -- and you wouldn't be able to see them in most situations against the black background

Reply:
I'm with Entrance here. The scoreboard needs a redo anyways, so for me it is fine to use these icons in there. I see no point in making the others work just to redo it at a later point again

Reply:
i might have expressed myself poorly.

I think the original goal of "correlating miniscoreboard and scoreboard" is good, but currently not really achievable as the two scoreboards currently differ quite a lot (hence the pushback to the scoreboard rework).

But i still think the current PR would solve another issue with slight modifications. As Entrance already said:
> On the other hand, reserving the icons to EXCLUSIVELY mean that some level of bodysearch information is available will drastically cut down the time players spend folding and unfolding scoreboard data, and all the tooltips could be amended to say "Click to expand bodysearch info" or something. (if it were an imgbutton, right now it can't pass-through doclick)

This would probably boil down to removing the 'indirect confirmation'/'killlist' icon and only using the other two. Additionally changing the icon descriptions to something like "(Dete) Body search results available" should also be done.
(Maybe find something shorter here, i didn't think a lot about the specific description)


TLDR:
- Correlation of Mini-scoreboard and Scoreboard should be thought about in a scoreboard rework
- Use the changes now to convey "Hey there are body search results available for this confirmed dead person"

Reply:
i don't have a great deal of time to continue working on this one in particular, if precise suggestions to the code are made i can change them and test it but i can't really determine what is desired to be changed from this at the moment